config/config.example.toml

```
@@ -194,6 +194,7 @@ gocardless.base_url = "https://api-sandbox.gocardless.com"
 helcim.base_url = "https://api.helcim.com/"
 iatapay.base_url = "https://sandbox.iata-pay.iata.org/api/v1"
 klarna.base_url = "https://api-na.playground.klarna.com/"
+mifinity.base_url = "https://demo.mifinity.com"
```

```suggestion
mifinity.base_url = "https://demo.mifinity.com/"
```

Put a `/` in the end at all the places you defined base url